pkgs/development/python-modules/langgraph-checkpoint/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildPythonPackage,
+  dataclasses-json,
+  fetchFromGitHub,
+  langchain-core,
+  langgraph-sdk,
+  poetry-core,
+  pytest-asyncio,
+  pytestCheckHook,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "langgraph-checkpoint";
+  version = "1.0.3";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "langchain-ai";
+    repo = "langgraph";
+    rev = "refs/tags/checkpoint==${version}";
+    hash = "sha256-5JP9f2uHNo71btQ96sBPlS7JPqo35C3VEMeHN1cJSro=";
+  };
+
+  sourceRoot = "${src.name}/libs/checkpoint";
+
+  build-system = [ poetry-core ];
+
+  dependencies = [ langchain-core ];
+
+  pythonImportsCheck = [ "langgraph.checkpoint" ];
+
+  nativeCheckInputs = [
+    dataclasses-json
+    pytest-asyncio
+    pytestCheckHook
+  ];
+
+  passthru = {
+    updateScript = langgraph-sdk.updateScript;
+  };
```

Comment:
```suggestion
```
Note that this updateScript does not update `langgraph-checkpoint`, so r-ryantm will not work.

Reply:
Fixed, I removed it.